CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ### Improvements
 
+- fix(secgroups): do not null out ingress & egress
+  [#128](https://github.com/pulumi/pulumi-eks/pull/128)
+    - Note: This PR reverses the default null values used for the
+      ingress and egress in-line rules of the secgroups, introduced in `v0.18.3`.
+      The null default was required to move to standalone secgroup rules, but it
+      has introduced [issues](https://github.com/pulumi/pulumi-eks/issues/127), and thus is being removed in this PR.
+    - Upgrade Path - This is a **breaking change** depending on the version used:
```

Comment:
Perhaps note that this is a breaking change **unless** you follow these steps?

Reply:
Done.